roles/openshift_excluder/tasks/status.yml

```diff
@@ -34,25 +34,37 @@
   - not openshift.common.is_containerized | bool
 
 - name: Record excluder status
-  command: "{{ openshift.common.service_type }}-excluder"
+  command: "{{ openshift.common.service_type }}-excluder status"
   register: excluder_status
   when:
   - "{{ openshift_excluder_installed.installed_versions | default([]) | length > 0 }}"
   - not openshift.common.is_containerized | bool
   failed_when: false
 
 - name: Record docker excluder status
-  command: "{{ openshift.common.service_type }}-docker-excluder"
+  command: "{{ openshift.common.service_type }}-docker-excluder status"
   register: docker_excluder_status
   when:
   - "{{ docker_excluder_installed.installed_versions | default([]) | length > 0 }}"
   - not openshift.common.is_containerized | bool
   failed_when: false
 
+# Unfortunately rpm_q doesn't return installed_versions if there aren't any installed
```

Comment:
Have we shipped a version of the excluder with the status command yet? If not, I think it would be better to handle this in the excluder rather than working around it here.